### PR TITLE
added cid

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: cid-redirects.json [hunk 1]
```diff
@@ -1404,6 +1404,7 @@
   "/cid/1958": "/docs/integrations/amazon-aws/security-hub",
   "/cid/1959": "/docs/integrations/containers-orchestration/vmware",
   "/cid/1960": "/docs/integrations/saas-cloud/pagerduty-v2",
+  "/cid/2122": "/docs/integrations/saas-cloud/gmail-tracelogs",
```
Comment (kimsauce) [inline cid-redirects.json:1407]: This will work once you merge https://github.com/SumoLogic/sumologic-documentation/pull/1091

Comment (JainM6) [reply cid-redirects.json:1407]: Thanks, @kimsauce:  I just merged #1091, but this is still failing.

Comment (JainM6) [reply cid-redirects.json:1407]: @kimsauce Thanks Kim, it is merged now.

Comment (kimsauce) [reply cid-redirects.json:1407]: You have to push this button first to update this build. It succeeded after I did that:

Screen Shot 2022-12-02 at 10 54 35 AM

Comment (JainM6) [reply cid-redirects.json:1407]: Yeah I saw that. Thank you so much @kimsauce.

**Pull request merged**